gulpfile.ts

```diff
@@ -159,6 +160,22 @@ function writeFilePromise(path, content) {
   });
 }
 
+function sitemap(cards: SingleCardJSON[]): SingleCardJSON[] {
+  const baseurl = 'http://www.aliemcards.com';
+  let sitemap = `${baseurl}\n`;
```

Comment:
I like that better. Didn't think about using a function call within the string declaration. But I guess the `/`` is just a disguised function call.